vendor/k8s.io/kubernetes/pkg/util/iptables/iptables.go

```diff
@@ -134,20 +128,19 @@ const LockfilePath16x = "/run/xtables.lock"
 type runner struct {
 	mu              sync.Mutex
 	exec            utilexec.Interface
-	dbus            utildbus.Interface
 	protocol        Protocol
 	hasCheck        bool
+	hasListener     bool
 	waitFlag        []string
 	restoreWaitFlag []string
 	lockfilePath    string
 
 	reloadFuncs []func()
-	signal      chan *godbus.Signal
```

Comment:
Also in "UPSTREAM 81517: Add iptables.Monitor, use it from kubelet and kube-proxy", `reloadFuncs` (and the blank line) should go away too

Reply:
done